esp-wifi/src/wifi/mod.rs

```diff
@@ -13,22 +13,40 @@ use core::{
 
 use critical_section::{CriticalSection, Mutex};
 use enumset::{EnumSet, EnumSetType};
-#[cfg(feature = "sniffer")]
-use esp_wifi_sys::include::{
-    esp_wifi_80211_tx,
-    esp_wifi_set_promiscuous,
-    esp_wifi_set_promiscuous_rx_cb,
-    wifi_promiscuous_pkt_t,
-    wifi_promiscuous_pkt_type_t,
-};
 use esp_wifi_sys::include::{
```

Comment:
I'm wondering if there is any downside to just glob importing `esp_wifi_sys::*`?

Reply:
I thought not but it triggers a lot of ambiguous-imports errors - probably worth to address some day but not sure